Makefile

```
@@ -1119,13 +1119,15 @@ pkg: $(PKG)
 
 .PHONY: corepack-update
 corepack-update:
-	rm -rf /tmp/node-corepack-clone
-	git clone 'https://github.com/nodejs/corepack.git' /tmp/node-corepack-clone
-	cd /tmp/node-corepack-clone && yarn pack
-	rm -rf deps/corepack && mkdir -p deps/corepack
-	cd deps/corepack && tar xf /tmp/node-corepack-clone/package.tgz --strip-components=1
+	mkdir -p /tmp/node-corepack
+	wget -q "$$(npm view corepack dist.tarball)" -O /tmp/node-corepack/package.tgz
```

Worth noting that `wget` is not available by default on macOS, while `curl` is. Not sure if it matters here though, feel free to ignore.
```suggestion
	curl -qLo /tmp/node-corepack/package.tgz "$$(npm view corepack dist.tarball)"
```